bundles/org.openhab.binding.luxom/src/main/java/org/openhab/binding/luxom/internal/handler/LuxomDimmerHandler.java

```diff
@@ -0,0 +1,176 @@
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.luxom.internal.handler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.luxom.internal.LuxomBindingConstants;
+import org.openhab.binding.luxom.internal.handler.config.LuxomThingDimmerConfig;
+import org.openhab.binding.luxom.internal.handler.util.LocalizationService;
+import org.openhab.binding.luxom.internal.handler.util.PercentageConverter;
+import org.openhab.binding.luxom.internal.protocol.LuxomAction;
+import org.openhab.binding.luxom.internal.protocol.LuxomCommand;
+import org.openhab.core.library.types.IncreaseDecreaseType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link LuxomDimmerHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Kris Jespers - Initial contribution
+ */
+@NonNullByDefault
+public class LuxomDimmerHandler extends LuxomThingHandler {
+    private final Logger logger = LoggerFactory.getLogger(LuxomDimmerHandler.class);
+
+    public LuxomDimmerHandler(Thing thing, LocalizationService localizationService) {
+        super(thing, localizationService);
+    }
+
+    @Nullable
+    private LuxomThingDimmerConfig config;
+    private final AtomicReference<Integer> lastLightLevel = new AtomicReference<>(0);
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        config = getThing().getConfiguration().as(LuxomThingDimmerConfig.class);
+
+        logger.debug("Initializing Switch handler for address {}", getAddress());
+
+        initDeviceState();
+    }
+
+    @Override
+    protected void initDeviceState() {
+        logger.debug("Initializing device state for Switch {}", getAddress());
+        @Nullable
+        Bridge bridge = getBridge();
+        if (bridge == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
+        } else if (ThingStatus.ONLINE.equals(bridge.getStatus())) {
+            if (config != null && config.doesNotReply) {
+                logger.warn("Switch {} will not reply, so always keeping it ONLINE", getAddress());
```

Comment:
I guess no reason to warn about this if configured deliberately because of this issue?